modules/apigee_edge_teams/src/Entity/ListBuilder/TeamAppListByTeam.php

```diff
@@ -60,8 +63,8 @@ class TeamAppListByTeam extends AppListBuilder implements ContainerInjectionInte
    * @param \Drupal\Core\Routing\RouteMatchInterface $route_match
    *   The route match object.
    */
-  public function __construct(EntityTypeInterface $entity_type, EntityTypeManagerInterface $entity_type_manager, RendererInterface $render, RequestStack $request_stack, TimeInterface $time, RouteMatchInterface $route_match) {
-    parent::__construct($entity_type, $entity_type_manager, $render, $request_stack, $time);
+  public function __construct(EntityTypeInterface $entity_type, EntityTypeManagerInterface $entity_type_manager, ConfigFactoryInterface $config_factory, RendererInterface $render, RequestStack $request_stack, TimeInterface $time, RouteMatchInterface $route_match) {
+    parent::__construct($entity_type, $entity_type_manager, $config_factory, $render, $request_stack, $time);
```

Comment:
This is a breaking change. We should add the `$config_factory` as the last and optional parameter, and instantiate the service if missing during the construct call. See `UserController::_construct()` as an example: https://git.drupalcode.org/project/drupal/-/blob/8.8.x/core/modules/user/src/Controller/UserController.php#L60-97